webapp/app/.server/service/protocol-upgrade-handler-abi.ts

```diff
@@ -772,9 +772,9 @@ const PROTOCOL_UPGRADE_HANDLER_RAW_ABI: Abi = [
     ],
     anonymous: false,
   },
-];
+] as const;
```

Comment:
There 3 different strategies for these the 3 abis.
This is one is "as const"
the second is none
The third one is "satisfies ABI"

Reply:
Fixed